src/tabset/tabset.ts

```diff
@@ -91,40 +90,36 @@ export interface NgbTabChangeEvent {
   preventDefault: () => void;
 }
 
+
 /**
  * A component that makes it easy to create tabbed interface.
  */
 @Component({
   selector: 'ngb-tabset',
   exportAs: 'ngbTabset',
   template: `
-    <ul [class]="'nav nav-' + type + (orientation == 'horizontal'?  ' ' + justifyClass : ' flex-column')" role="tablist">
```

Comment:
this removes aria roles and attrs while the nested content roles are kept (role="tabpanel", etc), was this indended?

Reply:
it's `<ul → ngbNav ← ...>` that sets the `tablist` role now, if I understand your comment correctly. Same for `ngbNavLink` setting `tab` role and `ngbNavOutlet` setting `tabpanel` role